Python/Product/Analysis/ModuleResolver.cs

```diff
@@ -112,7 +112,7 @@ bool absoluteImports
             // Handle relative module names
             if (relativeModuleName.FirstOrDefault() == '.') {
                 var fullName = GetModuleFullName(importingFrom, relativeModuleName);
-                if (fullName.Any(c => c != '.')) {
+                if (fullName.StartsWithOrdinal(".") || fullName.EndsWithOrdinal(".")) {
```

Comment:
Invert the sense of this check. We _don't_ want to return the name if it hasn't been resolved (still starts/ends with dots).